ical.js

```diff
@@ -298,8 +298,13 @@ const dateParameter = function (name) {
         const vTimezone =
           Object.values(stackItemWithTimeZone).find(({type}) => type === 'VTIMEZONE');
 
-        newDate = vTimezone && moment.tz.zone(vTimezone.tzid) ?
-          moment.tz(value, 'YYYYMMDDTHHmmss', vTimezone.tzid).toDate() :
+        // if the VTIMEZONE contains multiple TZIDs, use the last one in order
+        const normalizedTzId = vTimezone ? 
+          typeof vTimezone.tzid == 'object' ? vTimezone.tzid[vTimezone.tzid.length - 1] : vTimezone.tzid :
```

Comment:
If you want to check if `vTimezone.tzid` is an array it's safer to do `Array.isArray(vTimezone.tzid)` instead of `typeof vTimezone.tzid == 'object'`. Every array is an object but not every object is an array.

Reply:
Right - will fix.